src/main/java/nl/tudelft/pixelperfect/gamestates/PlayState.java

```diff
@@ -118,4 +118,9 @@ public GameState handleState() {
     }
     return this;
   }
+
+  @Override
+  public boolean isRunning() {
+    return true;
+  }
```

Comment:
Is it possible to create a PlayState and call isRunning() while the game is not running?